crates/typst/src/math/row.rs

```diff
@@ -161,7 +161,11 @@ impl MathRun {
         if self.0.len() == 1 {
             self.0.into_iter().next().unwrap()
         } else {
-            FrameFragment::new(ctx, styles, self.into_frame(ctx, styles)).into()
+            let text_like =
+                self.iter().all(|e| e.math_size().is_none() || e.is_text_like());
```

Comment:
why `e.math_size().is_none()`?

Reply:
Because it might be a Space, which shouldn't affect the layout but if it's not excluded, it will make an otherwise true result false.

Reply:
ah okay, so it is not related to the MathSize, it's more a convenient way to check whether it is `Glyph | Variant | Frame`? if yes, that's not immediately clear.

Reply:
Yes, but you can also think that not having a MathSize will make it not clear how much space the fragment shall occupy in the vertical dimension and so its text_like() is meaningless.

Reply:
Could you add a brief comment above the let binding?

Reply:
Done :)